src/chigame/games/templatetags/tags.py

```diff
@@ -0,0 +1,11 @@
+from django import template
+
+from chigame.games.models import Game
+
+register = template.Library()
+
+
+@register.simple_tag
+def get_games():
```

Comment:
Similar to above, add a comment explaining why this code is here and what purpose it serves.